libcontainer/container_linux.go

```diff
@@ -228,6 +228,14 @@ func (c *linuxContainer) Set(config configs.Config) error {
 }
 
 func (c *linuxContainer) Start(process *Process) error {
+	// configure linux personality before starting the process
```

Comment:
why is this done before the lock? (right below)

Reply:
I can move this below the lock that should do no harm but idea was to change execution domain before anything happens for the process.